server/supernova/sc/sc_osc_handler.cpp

```diff
@@ -203,16 +204,16 @@ template <typename T> static inline void consume(T&& object) {
 }
 
 void send_done_message(endpoint_ptr const& endpoint, const char* cmd) {
-    char buffer[128];
-    osc::OutboundPacketStream p(buffer, 128);
+    char buffer[1024];
+    osc::OutboundPacketStream p(buffer, 1024);
     p << osc::BeginMessage("/done") << cmd << osc::EndMessage;
 
     endpoint->send(p.Data(), p.Size());
 }
 
 void send_done_message(endpoint_ptr const& endpoint, const char* cmd, osc::int32 index) {
-    char buffer[128];
-    osc::OutboundPacketStream p(buffer, 128);
+    char buffer[1024];
+    osc::OutboundPacketStream p(buffer, 1024);
```

Comment:
I would rather see this coded into a single variable that describes what this size means, and use that variable in place of coding 1024 into these places directly. That way, in the future if there is a change like this, it can be done in one location.

Reply:
I see what you mean. 1024 is just one of several OSC message buffer sizes that Supernova uses throughout the code. I've spotted 128, 1024, 2048, 4096 and some variable length buffers. Of course, it would be nice to have const variables instead of magic numbers, but I fear this is a seperate project...